lib/url.js

```diff
@@ -87,7 +87,7 @@ function urlParse(url, parseQueryString, slashesDenoteHost) {
 
 Url.prototype.parse = function(url, parseQueryString, slashesDenoteHost) {
   if (typeof url !== 'string') {
-    throw new TypeError("Parameter 'url' must be a string, not " + typeof url);
+    throw new TypeError('Parameter "url" must be a string, not ' + typeof url);
```

Comment:
I think you missed the string end :) I just switched double quotes with singe quotes and viceversa